build/images/scripts/start_ovs

```diff
@@ -109,6 +109,8 @@ SLEEP_PID=
 
 function quit {
     log_info $CONTAINER_NAME "Stopping OVS before quit"
+    # sleep 2 seconds to wait antrea-agent shutdown for Flexible-IPAM
+    sleep 2
```

Comment:
is a static wait like the above enough? Not sure if you plan to iterate in a followup PR to ensure RestoreOVSBridge completes before stopping ovs.

Reply:
2s is unreliable. We saw many flakes when relying on arbitrary numbers got from test environment. You cannot see antrea-agent process in antrea-ovs container. I agree with @salv-orlando it should check the configuration instead. Whatever you want to wait for antrea-agent to do, you should check it here with a safe timeout (in case agent crashed and is not going to do it)

Reply:
Added uplink port check here and wait up to 10s.